Corona/Service/JHURepoDataService.swift

```diff
@@ -159,7 +161,7 @@ public class JHURepoDataService: DataService {
 			var series: [Date: Statistic] = [:]
 			for column in confirmedTimeSeries.values.indices {
 				let dateString = dateStrings[dateStrings.startIndex + column]
-				if let date = dateFormatter.date(from: dateString) {
+				if let dateEntry = mappedDates[dateString], let date = dateEntry {
```

Comment:
I'm sorry, but can't we access the date directly this way (without creating the temp dictionary):
`if let date = dateValues[dateValues.startIndex + column]`

Reply:
We could, but if `dateStrings` contains one invalid date it means that `dateValues` will have one less value.
```swift
dateStrings = ["20/10/21", "20/10/20", "invalid", "20/10/18"]
let dateValues = dateStrings.map { dateFormatter.date(from: $0) } // [20-10-21, 20-10-20, 20-10-18]
// columns will still be equal to 4 and when iterating by index we'll go out of bounds
```

Reply:
Indeed, I'll update the PR now

Reply:
Sorry, I got confused 😅